.github/workflows/ubuntu.yml

```diff
@@ -13,11 +13,29 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Checkout submodules
+      run: git submodule update --init --recursive
 
     - name: Install libmicrohttpd
       run: |
-        sudo apt-get update -qq 
+        sudo apt-get update -qq
         sudo apt-get install libmicrohttpd-dev
 
     - name: Build Faust
-      run: make
\ No newline at end of file
+      run: make
+
+    - name: Copy c header to expected path by impulse test
+      run: |
+         mkdir -p faust/dsp/
+         cp compiler/generator/libfaust.h faust/dsp/
+
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+
+    - name: Run impulse tests (C++)
+      run: cd tests/impulse-tests && make travis
```

Comment:
Strange that the target is called "travis" (tests don't run on Travis CI, no?). But of course that's somewhat independent of this PR, if it happens to be called like this already.

Reply:
It is a subset of the cpp tests that finishes in below 1h